packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -43,10 +43,10 @@ type IndexStatusEvent = {
   error?: Error
 }
 type IndexStatusCallback = (event: IndexStatusEvent) => Promise<void>
-type OnPutCallback = (key: string, value: any) => Promise<void>
-type OnDeleteCallback = (key: string) => Promise<void>
+export type OnPutCallback = (key: string, value: any) => Promise<void>
+export type OnDeleteCallback = (key: string) => Promise<void>
 
-type CreateDatabase = {
+export type CreateDatabase = {
```

Comment:
why are we exporting these? I don't think they need to be

Reply:
Just in case someone wants to do.

```ts
const githubOnPut : OnPutCallback = () =>{

//...
}
```